lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -859,7 +859,7 @@ def signature_of(method)
           nil
         end
 
-        sig { params(constant: Module).returns(String) }
+        sig { params(constant: T::Types::Base).returns(String) }
         def type_of(constant)
```

Comment:
Actually I wonder if we should move this method as a service of `T::Types::Base` since we monkey patch it now?

Reply:
We don't yet monkey-patch it, but yes, that's a good idea. I'll do that once I've merged the first PR that starts the monkey-patching business.